docs/docs/20-usage/75-project-settings.md

```diff
@@ -41,11 +41,19 @@ Only server admins can set this option. If you are not a server admin this optio
 
 ## Custom trusted clone plugins
 
-The clone step may require Git credentials (e.g. for private repos) which are injected via `netrc`.
+During the clone process, Git credentials (e.g., for private repositories) may be required.
+These credentials are provided via [`netrc`](https://everything.curl.dev/usingcurl/netrc.html).
 
-They are only injected into trusted plugins listed in the env var `WOODPECKER_PLUGINS_TRUSTED_CLONE` or in this repo setting.
+These credentials are injected only into trusted plugins specified in the (admin) environment variable `WOODPECKER_PLUGINS_TRUSTED_CLONE` or declared in this repository-level setting.
```

Comment:
IMO the admin part is a bit confusing. Drop it or use `server environment variable`? 

```suggestion
These credentials are injected only into trusted plugins specified in the environment variable `WOODPECKER_PLUGINS_TRUSTED_CLONE` or declared in this repository-level setting.
```

Reply:
Reading this part as a new user, it is not clear that this is an instance-wide env var that users cannot set themselves in a build. Why is "server environment variable" more clear?

Reply:
Because this env var need to be set in the server.

Reply:
Hm. OK I see why "admin" env var is not fully clear. In my mind I refer to everything that is an instance-wide config (no matter if server or agent) as an "admin task" or "admin env var".
The WP docs also bundle that below "administration".

But tl;dr - is the rephrased option acceptable?

Reply:
Works for me